src/caret_analyze/plot/visualize_lib/bokeh/util/hover.py

```diff
@@ -208,6 +218,9 @@ def to_list(self) -> list[str]:
                           'publish_node_name', 'subscribe_node_name']
         elif isinstance(self._target_object, (Publisher, Subscription)):
             hover_keys = ['legend_label', 'node_name', 'topic_name']
+        elif isinstance(self._target_object, Path):
+            # hover_keys = ['legend_label']
```

Comment:
Delete unnecessary comments.

Reply:
Fixed in https://github.com/tier4/CARET_analyze/commit/f4cd2af6e1f243a8c68186f3341424cb97eae919